### PR TITLE
ENH: add TIFF handler

### PR DESCRIPTION
**Pull request by CJ-Wright**
Add handler for TIFF files

### CODE REVIEW

**Comment by tacaswell** [comment]
What is wrong with `AreaDetectorTiffHandle`?

**Comment by CJ-Wright** [comment]
I guess I am a little bit confused by the `point_number` logic. I just have single frame tiff images and I just want them to open.

File: filestore/handlers.py [hunk 1]
```diff
@@ -436,3 +436,13 @@ def __call__(self, frame_no):
 
     def get_file_list(self, datum_kwarg_gen):
         return [self._fpath]
+
+
+class TiffHandler(HandlerBase):
```
Comment (tacaswell) [inline filestore/handlers.py]: Can you all this `SingleTiffHandler` or something similar?

File: filestore/handlers.py [hunk 2]
```diff
@@ -436,3 +436,13 @@ def __call__(self, frame_no):
...
+    specs = {'TIFF'} | HandlerBase.specs
+
+    def __init__(self, name):
```
Comment (tacaswell) [inline filestore/handlers.py]: Can you change name -> filename to math the rest of the handlers?

**Comment by tacaswell** [comment]
power cycled to pick up fixes on master for the CI setup

**Comment by CJ-Wright** [comment]
Thanks!

**Comment by tacaswell** [comment]
I am a tad concerned about grabbing "TIFF" as the key for this, we make steal that from you in the future.

**Comment by CJ-Wright** [comment]
Do you have a different name in mind?

**Comment by tacaswell** [comment]
I like 'SINGLE_TIFF'

**Comment by CJ-Wright** [comment]
Do you want me to revert the PR and add SINGLE_TIFF

**Comment by tacaswell** [comment]
just open a new PR and don't worry about reverting.

**Pull request merged**